### PR TITLE
Validate a CR update with all the operators

### PR DESCRIPTION
**Pull request by tiraboschi**
Before really propagating an update from HCO CR
to all the managed operators,
try to validate it with all the operators in
dry-run mode. If just one of them refuses
the update, refuse as well and report it to the user
with a clear error message.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:
```release-note
Validate a CR update with all the operators
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by ovirt-infra** [comment]
All tests passed

File: pkg/webhooks/webhooks.go [hunk 1]
```diff
@@ -74,14 +82,44 @@ func (wh WebhookHandler) updateOperatorCr(ctx context.Context, hc *v1beta1.Hyper
 
 	switch obj := exists.(type) {
 	case *kubevirtv1.KubeVirt:
-		existingKv := obj
+		existing := obj
```
Comment (nunnatsa) [inline pkg/webhooks/webhooks.go:85]: Actually, we don't need this variable. But I wouln't hold the PR for that. (same note for all the cases)

**Review by nunnatsa — APPROVED** [review]
/approve

**Pull request merged**